src/docs/modules/ROOT/pages/meta-model-elements.adoc

```diff
@@ -22,6 +22,8 @@ image::aspect-meta-model.svg[width=100%]
 
 * An _Entity_ is a logical encapsulation of multiple values. It has a number of Properties.
 
+* An _Abstract Entity_ is a logical encapsulation of a shared concept.
```

Comment:
Why is "extends" not part of aspect-meta-model.svg? And also as relationship between AbstractEntity and Entity. I think it is missing and should also be mentioned in text.
e.g.

An entity can extend another entity or abstract entity. An abstract entity can extend another abstract entity.

Reply:
I have updated the diagram and the documentation.